ZalikovkaC#/Program.cs

```diff
@@ -11,11 +11,11 @@ class Program
         static void Main(string[] args)
         {
             int k_1 = 6, k_2 = 5, k_3 = 4, k_4 = 4, k_5 = 4, k_6 = 3, k_7 = 2, k_8 = 2;
-            int VM,  INF, HIST, TCA, ECO, ENG, ENG_1, FIZ;
+            int VM, FZ, INF, HIST, TCA, ECO, ENG, ENG_1, FIZ;
```

Comment:
Can you give more accurate names for these variables? Use camelCase for the names.